newsrevenuehub/rev-engine
# [NRH-262-286] Datetime timezones

#### What's this PR do?
We were sending datetimes back to the API without timezones. Two bugs were fixed by sending datetimes set on the frontend back with timezones included:
1. 262-- donation page edit, published date time was coming back 4 hours behind the date you saved.
2. 286 -- from and to filters in the donations list were excluding many donations for the selected day.

#### How should this be manually tested?
1. Add a time to a published_date in the page editor. Save it. Check the published time again and it should not be different from what you saved.
2. Set some from and two filters. No donations from the dates set should be removed from the list.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No
